Gemfile

```diff
@@ -52,4 +46,4 @@ gem 'factory_girl', group: [:development, :test]
 gem 'valid_attribute', group: [:development, :test]
 gem 'shoulda-matchers', group: [:development, :test], require: false
 gem 'devise'
-gem 'foundation-rails'
+gem 'foundation-rails', '= 5.5.1.0'
```

Comment:
Clean up your Gemfile. Delete comments. Delete `jbuilder` since you are using `active_model_serializers`. They essentially do the same thing. Move any gem that has `group: [:development, :test]` to the block you have for `group :development, :test`. Remove `byebug` since you are using `pry-rails`. Move `pry-rails` to `group :development, :test`.